doc/lsst.daf.butler/index.rst

```diff
@@ -100,6 +100,12 @@ Python API reference
 
 .. automodapi:: lsst.daf.butler
    :no-main-docstr:
+   :skip: ButlerURI
+
+.. py:class:: lsst.daf.butler.ButlerURI(uri)
```

Comment:
@jonathansick This does work in the sense that some words appear, but it doesn't result in something that is clickable. Am I doing something wrong? The `:skip:` fixes the pipelines.lsst.io build problem so it's not important.

Reply:
What automodapi does, once it detects the available symbols in a namespace, is create rst pages that in turn contain `py:class` directives and so on. Then it creates an automodsumm in place of the automodapi while creates the click-able table of contents. I'm assuming that this ButlerURI documentation in `lsst.daf.butler` is only temporary, so I don't think it's worth the trouble to fake the look of automodsumm.

Also note that if there are existing links in the documentation to methods of ButlerURI, you'd technically want to add those methods to your doc for ButlerURI. You'd do this by putting `py:method` directives inside the scope of the `py:class` directive. If there aren't such links, I wouldn't worry about it.

Reply:
There aren't any links to methods, just the class. The docs do build fine with this change so I'll leave it. The plan is to do a global replacement of `ButlerURI` with `ResourcePath` so I'll leave it as is and merge this to solve the immediate problem of the pipelines.lsst.io build not working.